arm.cmake

```diff
@@ -1,10 +1,14 @@
 set(CMAKE_SYSTEM_NAME "Generic")
 set(CMAKE_SYSTEM_PROCESSOR "arm")
 
+set(LOCATION_PATH $ENV{HOMEBREW_PATH})
```

Comment:
I'm not a big fan of `HOMEBREW_PATH` in the `arm.cmake` file. What do you think about calling it `ARMROOT` or `GCCROOT` or something similar?

Reply:
Sure as mentioned in the opening comment, I do not expect this to get merged, just wanted to suggest a fix to a issue i found.